lib/new.js

```diff
@@ -102,18 +102,15 @@ function cleanupTemplate() {
 }
 
 /**
- * Finds all dependencies with a version specified as "latest", looks up the package in the global
- * NPM registry, and replaces it with the actual version number of the latest version of the package.
+ * Pass each dependency and it's requested version through the `install-latest` library.
+ * This supports any valid semver version, but will resolve to the exact version.
  * @param {*} dependencies The dependencies with versions to resolve.
  */
 async function setDependencyVersions(dependencies) {
   const packageNames = Object.keys(dependencies);
 
   const dependencyPromises = packageNames.map(
-    packageName => dependencies[packageName] === 'latest' ?
-      latestVersion(packageName) :
-      // Default to the specified version.
-      Promise.resolve(dependencies[packageName])
+    packageName => latestVersion(packageName, { version: dependencies[packageName] })
```

Comment:
This is messing with `peerDependencies`. Those should be left alone, in my opinion. This is what the result was:
<img width="291" alt="Screen Shot 2019-04-10 at 9 09 43 AM" src="https://user-images.githubusercontent.com/12497062/55881379-8bb6f980-5b70-11e9-977f-6237f62abed3.png">

Reply:
Good catch!  That behavior used to happen, but since I run all deps through the library and not just those requesting `latest`, we saw that behavior.  I've adjusted to not run through `peerDependencies` and updated the test to confirm it.